lib/agenda.js

```diff
@@ -705,13 +790,15 @@ function processJobs(extraJob) {
     // Go through each jobName set in 'Agenda.process' and fill the queue with the next jobs
     for (jobName in definitions) {
       if ({}.hasOwnProperty.call(definitions, jobName)) {
+        debug('queuing up job to process: [%s]', jobName);
         jobQueueFilling(jobName);
       }
     }
 
   } else if (definitions[extraJob.attrs.name]) {
 
     // Add the job to list of jobs to lock and then lock it immediately!
+    debug('extra job was passed in, immediate lock and execute to occur!');
```

Comment:
This may need to be reworded.

Remove `!` to stay consistent with other debug statements.